src/jsonSchema.ts

```diff
@@ -1,8 +1,8 @@
 import * as Ajv from 'ajv';
```

Comment:
When you add `esModuleInterop: true` to the `tsconfig.json`, `* as ` should be removed and all the type errors will disapear.

Reply:
<img width="995" alt="Screen Shot 2020-09-18 at 4 25 32 pm" src="https://user-images.githubusercontent.com/10513364/93563596-94e60000-f9cb-11ea-9d80-57e73d1bf1cc.png">